test/bulk_insert/worker_test.rb

```diff
@@ -159,11 +160,13 @@ class BulkInsertWorkerTest < ActiveSupport::TestCase
     worker.add greeting: "second"
```

Comment:
On this test I would follow something similar to:

https://github.com/jamis/bulk_insert/blob/4a69affacf9714293710fb84e391bad14dca6ac3/test/bulk_insert_test.rb#L34-L43

```
if Testing.connection.adapter_name =~ /\APost(?:greSQL|GIS)/i
      worker = BulkInsert::Worker.new(
      Testing.connection,
      Testing.table_name,
      'id',
      %w(greeting age happy created_at updated_at color),
      500,
      false,
      false,
      true
    )
    # put the rest of the test here
else
  assert_raise(ArgumentError) {     worker = BulkInsert::Worker.new(Testing.connection, Testing.table_name, 'id', %w(greeting age happy created_at updated_at color), 500, false, false, true) }
```

you can eventually also capture the error message as per the example linked and perform a partial matching on the error message